README.md

```diff
@@ -111,3 +111,74 @@ or a specific test can be run using:
+ok: [13.43.77.114]
+
+TASK [Install python] ***********************************************************************************************************************************************************************************************************************************************************************************
+ok: [13.43.77.114]
+
+TASK [Install node] *************************************************************************************************************************************************************************************************************************************************************************************
+ok: [13.43.77.114]
+
+TASK [Install poetry] ***********************************************************************************************************************************************************************************************************************************************************************************
+ok: [13.43.77.114]
+
+TASK [Create app folder] ********************************************************************************************************************************************************************************************************************************************************************************
+ok: [13.43.77.114]
+
+TASK [Checkout repo] ************************************************************************************************************************************************************************************************************************************************************************************
+changed: [13.43.77.114]
+
+TASK [Install poetry dependencies] **********************************************************************************************************************************************************************************************************************************************************************
+changed: [13.43.77.114]
+
+TASK [Install npm dependencies] *************************************************************************************************************************************************************************************************************************************************************************
+changed: [13.43.77.114]
+
+TASK [Build css] ****************************************************************************************************************************************************************************************************************************************************************************************
+changed: [13.43.77.114]
+
+TASK [Construct .env from template] *********************************************************************************************************************************************************************************************************************************************************************
+ok: [13.43.77.114]
+
```

Comment:
_Very_ minor, but I do tend to find my eyes just gloss over most of the expected output when something like this is here - would it be better just having the first few lines, then `...` and then the last few?